packages/eslint-plugin-orbit-components/__tests__/defaultTheme.test.ts

```diff
@@ -0,0 +1,81 @@
+import ruleTester from "../ruleTester";
+import defaultTheme, { ERRORS } from "../src/rules/defaultTheme";
+
+describe("defaultTheme", () => {
+  // @ts-expect-error TODO
```

Comment:
You can avoid a lot of these error suppression comments here and in the rule file itself, along with having to specify argument types, by simply setting type of the rule export itself to `Rule.RuleModule` type, which is what `ruleTester.run` expects:

```ts
// defaultTheme.ts
import * as t from "@babel/types";
import type { Rule } from "eslint";

// ...

const defaultThemeRule: Rule.RuleModule = {
  // ...
};

export default defaultThemeRule;
```

TypeScript will helpfully point out all suppression comments that are no longer needed, so you don't have to worry about missing something.

Unfortunately, one thing I wasn't able to figure out is the `never` problem in the `VariableDeclaration` function, it doesn't make sense to me, but maybe you'll be able to figure it out! 😉 🤞

Reply:
I was too lazy to solve it :D thanks for it